content/tutorials/first-aws-environment.md

```diff
@@ -0,0 +1,210 @@
+
+```bash
+atmos terraform backend generate component static-site --stack uw2-dev
+```
+
+Exactly the same as for our `tfstate-backend` component, our `atmos` `backend generate` command will put a `backend.tf.json` file in our `components/terraform/static-site/` directory so that component will always utilize the correct backend. If this were a real project that we were working on then we'd actually check those files into git so that our backend configuration would persist going forward, but since this is a tutorial we'll skip that step.
+
+### 3. Apply our Static Site
+
+Next, let's move onto applying our `static-site` component to actually build something real on AWS. Well that's pretty simple actually:
+
+```bash
+aws-vault exec $YOUR_PROFILE -- atmos terraform deploy static-site --stack uw2-dev
+```
+
+This command will plan and apply our `static-site` component with our configuration specified in our `stacks/uw2-dev.yaml` stack. This should output something similar to the following at the end:
+
+```txt
+Apply complete! Resources: 7 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+domain_name = "CLOUDFRONT_ID.cloudfront.net"
+s3_bucket_name = "acme-uw2-dev-static-site-RANDOM_ID-origin"
+```
+
+If you copy the `domain_name` value and paste it into your browser then you should see our simple static site!
+
+![Static Site Example Image](/assets/tutorial-3-static-site.png)
+
+As you can see, this deploys our static site for our `dev` environment, but not our `prod` environment. This tutorial won't cover deploying the `prod` environment, but feel free to play around with the variables if you would like to mess around with it!
+
+## Conclusion
+
+In this tutorial, we've given you a bit more of taste of what it looks like to work on a real project while following the SweetOps methodology:
+
+1. We've seen real world components: `tfstate-backend` and `static-site`. These are sharable root modules that you didn't need to configure or touch to get up and running, you just need to supply the configuration. We have a whole library of components (which `tfstate-backend` is one of) over at [`cloudposse/terraform-aws-components`](https://github.com/cloudposse/terraform-aws-components).
+1. We utilized `atmos` to deploy our terraform state backend for our various components and then also generate the `backend.tf.json` files to ensure our components are properly configured from a central configuration in our `stacks/` directory.
+1. You now have an example of what it looks like to organize stack configurations **for different environments and regions**. This is immensely important and is why SweetOps is so powerful: Our stacks centrally define our environments, where they're deployed, and how they're configured. In a normal engineering ecosystem that information is typically only ever in a few senior engineers heads, but with stacks we get it for free as we declare it as code.
```

Comment:
```suggestion
1. You now have an example of what it looks like to organize stack configurations **for different environments and regions**. This is immensely important and is why SweetOps is so powerful: Our stacks centrally define our environments, where they're deployed, and how they're configured; the components are completely agnostic to their environment and only responsible for business logic. This separation is critical to building reusable, composable infrastructure. Not to mention, we ensure that critical configurations are declared as code and not just committed to the memory of a few senior engineers.
```

Reply:
Thanks for the re-write -- That previous sentence was just bad.